doc/howtos/using_ubuntu_as_sos.rst

```diff
@@ -0,0 +1,184 @@
+# sudo apt-get install libssl-dev
+# sudo apt-get install vim
+# sudo apt-get install libsystemd-dev
+# sudo apt-get install libevent-dev
+# sudo apt-get install libxml2-dev
+
+Step #5, Checkout source code from open source project repositories
+# git clone https://github.com/projectacrn/acrn-hypervisor
+Note: if you encounter issues with “git clone”, you can download ZIP package directly. It’s so easy.
+
+Step # 6, building the ACRN hypervisor and device model source code on Ubuntu system
+Building device model and hypervisor from source trees on NUC you just setup. You can reference the section of Getting started guide 
+of the ACRN open source project, or following up below commands to build directly here. 
+# cd acrn-hypervisor
+# make PLATFORM=uefi
+The build results are found in the build directory.
+For example:
+# ls acrn-hypervisor/build
+devicemodel
+hypervisor
+tools
+
+Steps #7,  Preparing Service OS kernel 
+You can download latest Service OS kernel from below links:
+https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/
+
+for example:
+https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/linux-pk414-sos-4.14.41-39.x86_64.rpm
+or
+https://cdn.download.clearlinux.org/current/x86_64/os/Packages/linux-pk414-sos-4.14.41-39.x86_64.rpm
+
+# mkdir ~/kernel-build
+# cd ~/kernel-build
+# wget  https://download.clearlinux.org/releases/current/clear/x86_64/os/Packages/linux-pk414-sos-4.14.41-39.x86_64.rpm
+# sudo apt-get install rpm2cpio
+# rpm2cpio linux-pk414-sos-4.14.41-39.x86_64.rpm | cpio -idmv 
+# sudo cp -r ~/kernel-build/usr/lib/modules/4.14.41-39.pk414-sos/ /lib/modules/
+
+Step #8, Deploying the hypervisor
+Mount the EFI partition and verify you have the following files, then add the ACRN hypervisor and Clear Linux kernel to the EFI 
+Partition, 
+# umount /boot/efi 
+# lsblk
+# mount /dev/sda1 /mnt
+# ls /mnt/EFI/ubuntu
+  fw  fwupx64.efi  grub.cfg  grubx64.efi  MokManager.efi  shimx64.efi
+
+Copy acrn.efi  to /mnt/EFI/acrn/ 
+# sudo mkdir /mnt/EFI/acrn/
+# sudo cp acrn-hypervisor/build/acrn.efi /mnt/EFI/acrn	
+
+Step #9, Configure the EFI firmware to boot the ACRN hypervisor by default
+# sudo -s
+# efibootmgr -c -l "\EFI\acrn\acrn.efi" -d /dev/sda -p 1 -L "ACRN Hypervisor                                                                                                    NUC" -u "bootloader=\EFI\ubuntu\grubx64.efi"
+Then you can see the result of boot order. Also you can check the boot order with below command:
+# efibootmgr -v            	
+or setting boot order with this command to make sure the acrn.efi is the first boot option:
+# efibootmgr -o x, x       	
+Note: by default, the “ACRN Hypervisor” you just added should be the first one to boot. But please make sure it is, or select it 
+manually after press “F10” when system booting up.   
+
+Step #10, Deploy the SOS kernel
+# sudo cp ~/kernel-build/usr/lib/kernel/org.clearlinux.pk414-sos.4.14.41-39 /boot/acrn/
```

Comment:
This is the first time you use `/boot/acrn` so I expect it does not exist. Should it be created?

I'm also wondering if the target folder shouldn't really be somewhere under `/mnt/`, where the ESP is mounted at the moment (and the kernel needs to be somewhere on the ESP for Grub to be able to load it).